tests/verkle/eip4762_verkle_gas_witness/test_sstore.py

```diff
@@ -19,12 +19,13 @@
     WitnessCheck,
 )
 from ethereum_test_tools.vm.opcode import Opcodes as Op
+from ethereum_test_types.verkle.types import Hash
 
 # TODO(verkle): Update reference spec version
 REFERENCE_SPEC_GIT_PATH = "EIPS/eip-4762.md"
 REFERENCE_SPEC_VERSION = "2f8299df31bb8173618901a03a8366a3183479b0"
 
-TestAddress2Storage = {0: 0xAA, 1000: 0xBB}
+TestAddress2Storage: dict[int, Hash] = {0: Hash(0xAA), 1000: Hash(0xBB)}
```

Comment:
We can remove this later whenever the testing library supports both `int` and `Hash`.